ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-master-presubmits.yaml

```diff
@@ -1501,6 +1501,61 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-gcp-upgrade,?($|\s.*)
+  - agent: kubernetes
+    always_run: false
```

Comment:
I think we can make this always_run: true so we have some data on this, since I don't think most people will run hypershift e2e on their PRs. Unless you think this is too resource heavy